lib/reek/smells/smell_detector.rb

```diff
@@ -60,6 +63,12 @@ def enabled_for?(context)
         config.enabled? && config_for(context)[SmellConfiguration::ENABLED_KEY] != false
       end
 
+      def check_suppression(context)
+        return [] unless sniff(context).empty?
```

Comment:
The positive formulation is `if sniff(context).any?`.

I think @waldyr had a common `sniff(context)` first, but that became confusing.